_sass/blocks/revenues/_revenues_subpage_diving_graphics.scss

```diff
@@ -151,24 +155,58 @@
     .revenues_subpage-dyk_heading {
       background-color: $offshore-renewables-secondary-color;
     }
-    .revenues_subpage-steps_heading {
+    .revenues_subpage-steps_heading,
+    .revenues_subpage-steps_heading_more {
       color: $offshore-renewables-secondary-color;
     }
 
     @include respond-to(medium-up) {
-      background: url('../img/revenues/renew-offshore.svg') bottom left no-repeat;
-      background-size: 100%;
+      &:after {
+        background: url('../img/revenues/renew-offshore.svg') bottom left no-repeat;
+      }
+    }
+  }
+
+  &.offshore-renewables,
+  &.onshore-renewables {
+    @include respond-to(medium-up) {
+      position: relative;
+      z-index: 1;
+
+      &:before {
+        background: url('../img/revenues/renewables-bg-gradient.png') top left repeat-x;
+        background-size: contain;
+        bottom: 0;
+        content: '';
+        left: 0;
+        position: absolute;
+        right: 0;
+        top: 0;
+        z-index: -1;
+      }
+
+      &:after {
+        background-size: 100%;
+        bottom: 0;
+        content: '';
+        left: 0;
+        position: absolute;
+        right: 0;
+        top: 0;
+        z-index: 2;
+      }
     }
   }
 
   &.coal{
-    
+
     background: $coal-primary-color;
-    
+
     .revenues_subpage-dyk_heading {
       background-color: $coal-secondary-color;
     }
-    .revenues_subpage-steps_heading {
+    .revenues_subpage-steps_heading,
```

Comment:
Selector `revenues_subpage-steps_heading` should be written in lowercase with hyphens<br>Selector `revenues_subpage-steps_heading_more` should be written in lowercase with hyphens